sdk/documentintelligence/azure-ai-documentintelligence/README.md

```diff
@@ -213,6 +213,29 @@ from azure.core.credentials import AzureKeyCredential
 from azure.ai.documentintelligence import DocumentIntelligenceClient
 from azure.ai.documentintelligence.models import AnalyzeResult
 
+def get_words(page, line):
+    result = []
+    for word in page.words:
+        if _in_span(word, line.spans):
+            result.append(word)
```

Comment:
What if we just add this inline in the samples?

Reply:
You could consider doing the same for the other format_* methods

Reply:
I changed mostly all helper functions to call inline, there are one or two still need helper functions to make code to be clear.